interpreter/value.go

```diff
@@ -91,7 +91,7 @@ type Value interface {
 	// Stringer provides `func String() string`
 	// NOTE: important, error messages rely on values to implement String
 	fmt.Stringer
-	isValue()
+	//isValue()
```

Comment:
Had to make `interpreter.Value` extensible: Because with the vm would have it's own `FunctionValue` implementation (with compiled instructions as body).